docs/posts/2023-06/article3.md

```
@@ -115,7 +115,7 @@ order: 4
 
 “别这样，菲亚梅塔。”
 
-![](./res/illustration/LOFTER@左刀行（潮石镇2）.webp){style="float:left;max-width:40%;margin-right:1em"}
+![](./res/illustration/LOFTER@左刀行（潮石镇2）.webp){.image-left-float style="max-width: 40%;"}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)